feature_layers/display-route-layer/build.gradle

```
@@ -0,0 +1,88 @@
+plugins {
+    id 'application'
+    id 'org.openjfx.javafxplugin' // version is defined in settings.gradle
+}
+
+group = 'com.esri.samples'
+
+ext {
+    arcgisVersion = '200.0.0'
```

```suggestion
    arcgisVersion = '200.2.0-3933'
```
We should use an up to date version of the API. Kathleen can tell you all about daily builds! 😄

---

Updated arcgisVersion